src/screens/Customers/AddCustomer.tsx

```diff
@@ -4,43 +4,74 @@ import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 import { Redirect, RouteComponentProps, useHistory } from 'react-router-dom';
 import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
 import Button from '../../components/Button';
 import Checkbox from '../../components/Checkbox';
 import TextField from '../../components/TextField';
 import { SiteRecord } from '../../lib/airtable/interface';
 import { createCustomer } from '../../lib/airtable/request';
 import { setCurrentCustomerIdInRedux } from '../../lib/redux/customerData';
 import { selectAllTariffPlansArray } from '../../lib/redux/siteDataSlice';
-import { EMPTY_CUSTOMER } from '../../lib/redux/customerDataSlice';
+import { EMPTY_CUSTOMER, MeterType } from '../../lib/redux/customerDataSlice';
 import { selectCurrentSiteInformation } from '../../lib/redux/siteData';
+import TariffPlanCard from '../Profile/components/TariffPlanCard';
 
 const styles = (theme: Theme) =>
   createStyles({
-    content: {
-      color: theme.palette.text.primary,
-    },
     formControl: {
       width: '100%',
     },
+    inline: {
+      display: 'inline-flex',
+      width: '100%',
+    },
+    textContainer: {
+      padding: '2px 0px',
+    },
+    selectContainer: {
+      padding: '10px 0px',
+    },
+    nameInput: {
+      width: '130%',
+      paddingRight: '10px',
+    },
+    numberInput: {
+      width: '50%',
+    },
```

Comment:
This is done in `/base-components` (`BaseComponentsDemo.tsx`) more cleanly using flexbox.
![image](https://user-images.githubusercontent.com/35501399/114926343-175d6380-9de5-11eb-9846-43690e3a12a5.png)

The associated code for ^ is as follows

![image](https://user-images.githubusercontent.com/35501399/114926427-2cd28d80-9de5-11eb-9be9-11e1aeb60b67.png)

![image](https://user-images.githubusercontent.com/35501399/114926442-30feab00-9de5-11eb-86df-0c1196871221.png)


I think we should switch to the flexbox method as it's easier to interpret as well